# org_tensorflow 用の Android 向けパッチ

diff --git a/tensorflow/BUILD b/tensorflow/BUILD
index 6e6ab138a2f..8869f45aa5d 100644
--- a/tensorflow/BUILD
+++ b/tensorflow/BUILD
@@ -170,7 +170,7 @@ config_setting(
         {},
     ),
     values = if_oss(
-        {"crosstool_top": "//external:android/crosstool"},
+        {"crosstool_top": "@android_toolchain//:toolchain"},
         {},
     ),
     visibility = ["//visibility:public"],
@@ -184,7 +184,7 @@ config_setting(
     ),
     values = dict(
         if_oss(
-            {"crosstool_top": "//external:android/crosstool"},
+            {"crosstool_top": "@android_toolchain//:toolchain"},
         ),
         cpu = "x86",
     ),
@@ -199,7 +199,7 @@ config_setting(
     ),
     values = dict(
         if_oss(
-            {"crosstool_top": "//external:android/crosstool"},
+            {"crosstool_top": "@android_toolchain//:toolchain"},
         ),
         cpu = "x86_64",
     ),
@@ -214,7 +214,7 @@ config_setting(
     ),
     values = dict(
         if_oss(
-            {"crosstool_top": "//external:android/crosstool"},
+            {"crosstool_top": "@android_toolchain//:toolchain"},
         ),
         cpu = "armeabi",
     ),
@@ -274,7 +274,7 @@ config_setting(
     ),
     values = dict(
         if_oss(
-            {"crosstool_top": "//external:android/crosstool"},
+            {"crosstool_top": "@android_toolchain//:toolchain"},
         ),
         cpu = "armeabi-v7a",
     ),
@@ -289,7 +289,7 @@ config_setting(
     ),
     values = dict(
         if_oss(
-            {"crosstool_top": "//external:android/crosstool"},
+            {"crosstool_top": "@android_toolchain//:toolchain"},
         ),
         cpu = "arm64-v8a",
     ),
@@ -299,7 +299,7 @@ config_setting(
 config_setting(
     name = "android_mips",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "mips",
     },
     visibility = ["//visibility:public"],
@@ -308,7 +308,7 @@ config_setting(
 config_setting(
     name = "android_mips64",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "mips64",
     },
     visibility = ["//visibility:public"],
