diff --git a/BUILD.bazel b/BUILD.bazel
index 4ac8f10..eb5f21e 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -257,7 +257,7 @@ config_setting(
 config_setting(
     name = "android_armv7",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "armeabi-v7a",
     },
     visibility = ["//visibility:public"],
@@ -266,7 +266,7 @@ config_setting(
 config_setting(
     name = "android_arm64",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "arm64-v8a",
     },
     visibility = ["//visibility:public"],
@@ -275,7 +275,7 @@ config_setting(
 config_setting(
     name = "android_x86",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "x86",
     },
     visibility = ["//visibility:public"],
@@ -284,7 +284,7 @@ config_setting(
 config_setting(
     name = "android_x86_64",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "x86_64",
     },
     visibility = ["//visibility:public"],
